.editorconfig

```diff
@@ -1,2 +1,35 @@
-[*.{kt,kts}]
-disabled_rules=filename,no-wildcard-imports
+root = true
+
+[*]
+charset                   = utf-8
+end_of_line               = lf
+indent_size               = tab
+indent_style              = space
```

Comment:
Please refrain from aligning values. Even if it looks "ok" for the first time, it would look weird as new options appear or old ones are removed.

See https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment